src/main/java/com/google/cloud/tools/appengine/cloudsdk/process/StringBuilderProcessOutputLineListener.java

```diff
@@ -20,6 +20,7 @@
 public class StringBuilderProcessOutputLineListener implements ProcessOutputLineListener {
 
   // This used to be a StringBuilder but that wasn't thread safe.
+  @SuppressWarnings("JdkObsolete")
```

Comment:
should we just fix this warning?

Reply:
No, this is an overly aggressive warning for a case that doesn't apply here. StringBuffer is exactly what we need.